packages/theme/nuxt.config.js

```diff
@@ -3,10 +3,6 @@ import { getRoutes } from './routes';
 import theme from './themeConfig';
 
 export default {
-  server: {
```

Comment:
This is needed as well as vendure is working by default on port 3000. Please, revert this change